runtime/src/main/java/com/dylibso/chicory/runtime/TableInstance.java

```diff
@@ -0,0 +1,58 @@
+package com.dylibso.chicory.runtime;
+
+import com.dylibso.chicory.wasm.types.Limits;
+import com.dylibso.chicory.wasm.types.Table;
+import com.dylibso.chicory.wasm.types.Value;
+import com.dylibso.chicory.wasm.types.ValueType;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TableInstance {
+
+    private Table table;
+    private Map<Integer, Instance> instances;
```

Comment:
A Table holds function references, but we should invoke the functions on the Instance they belong to.